src/routes/login/+page.svelte

```diff
@@ -15,14 +19,16 @@
 <p>Please input your credentials</p>
 
 <form action="?/login" method="post">
-	<div>
+	<div class="content">
 		<label for="username">Name:</label>
```

Comment:
These labels look really ugly.
I think the placeholder is enough for the user to know what to put here.